Libraries/oneTBB/tbb-async-sycl/src/tbb-async-sycl.cpp

```diff
@@ -0,0 +1,195 @@
+//==============================================================
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+#include <cmath>  //for std::ceil
+#include <array>
+#include <atomic>
+#include <iostream>
+#include <thread>
+
+#include <CL/sycl.hpp>
+
+#include <tbb/blocked_range.h>
+#include <tbb/flow_graph.h>
+#include <tbb/global_control.h>
+#include <tbb/parallel_for.h>
```

Comment:
Need to include the following header file 'include "dpc_common.hpp"'
// dpc_common.hpp can be found in the dev-utilities include folder.
// e.g., $ONEAPI_ROOT/dev-utilities//include/dpc_common.hpp

please refer to simple_add sample or mandelbrot for the usage of the dpc_common.hpp